tools/pipelines/test-real-service-stress.yml

```diff
@@ -24,11 +24,15 @@ variables:
 - name: testWorkspace
   value: $(Pipeline.Workspace)/test
 
+lockBehavior: sequential
 stages:
-    # stress tests odsp
+  # stress tests odsp
   - stage:
     displayName:  Stress tests - Odsp
     dependsOn: []
+    # use a variable group with exclusive lock force only one run at a time and avoid overloading the server/throttling
+    variables:
```

Comment:
i don't understand what this is doing. from the docs i don't see mention of needing a variable group, or any interaction with one

Reply:
The exclusive-lock requirement is based on resources. From the [docs](https://learn.microsoft.com/en-us/azure/devops/pipelines/process/approvals?view=azure-devops&preserve-view=true&tabs=check-pass#exclusive-lock) at the top:
`Checks can be configured on environments, service connections, repositories, variable groups, secure files, and agent pools.` 

By default, the exclusive lock behavior is `runLatest`, but we want `sequential`.